Source/player.cpp

```diff
@@ -3601,12 +3601,7 @@ void ModifyPlrStr(int p, int l)
 	auto &player = Players[p];
 
 	int max = player.GetMaximumAttributeValue(CharacterAttribute::Strength);
-	if (player._pBaseStr + l > max) {
-		l = max - player._pBaseStr;
-	}
-	if (player._pBaseStr + l < 0) {
-		l = 0;
-	}
+	l = clamp(l, 0, max - player._pBaseStr);

```

Comment:
Might as well inline `max` now

Reply:
`inline int GetMaxStat(Player &player, CharacterAttribute attribute)
{
	return player.GetMaximumAttributeValue(attribute);
}`
Like this?